// config/rise.js
export const rise = {
  api_url: process.env.API_URL || 'https://api.sandbox.rise.store',
  api_version: process.env.API_VERSION || 'v1',
  public_key: process.env.RISE_API_PUBLIC || 'pk_test_c4beb40b-1651-48ef-a821-c28a81e98ade',
  channel_uuid: process.env.RISE_DEFAULT_CHANNEL || 'c034ad64-87c9-479c-ae5e-8c9aedf0b1f3',
  default_channel: process.env.RISE_DEFAULT_CHANNEL || 'c034ad64-87c9-479c-ae5e-8c9aedf0b1f3',
  live_mode: process.env.RISE_LIVE_MODE || false,

  // CAMPAIGNS
  primary_navigation_handle: process.env.RISE_PRIMARY_NAVIGATION_HANDLE || 'primary-navigation',
  featured_handle: process.env.RISE_FEATURED_HANDLE || 'featured',
  on_sale_handle: process.env.RISE_ON_SALE_HANDLE || 'on-sale',
  best_rated_handle: process.env.RISE_BEST_RATED_HANDLE || 'best-rated',
  deals_of_the_week_handle: process.env.RISE_DEALS_OF_THE_WEEK_HANDLE || 'deals-of-the-week',
  popular_categories_handle: process.env.RISE_POPULAR_CATEGORIES_HANDLE || 'popular-categories',
  hot_new_arrivals_handle: process.env.RISE_HOT_NEW_ARRIVALS_HANDLE || 'hot-new-arrivals',
  hot_new_arrivals_featured_handle: process.env.RISE_HOT_NEW_ARRIVALS_FEATURED_HANDLE || 'hot-new-arrivals-featured',
  hot_best_sellers_handle: process.env.RISE_HOT_BEST_SELLERS_HANDLE || 'hot-best-sellers',
  hot_best_sellers_featured_handle: process.env.RISE_HOT_BEST_SELLERS_FEATURED_HANDLE || 'hot-best-sellers-featured',
  trending_sellers_handle: process.env.RISE_HOT_TRENDING_SELLERS_HANDLE || 'trending-sellers',
  brands_handle: process.env.RISE_BRANDS_HANDLE || 'offer-brands',
}
